template/CONTRIBUTING.md.jinja

```diff
@@ -1,18 +1,18 @@
-## Contributing to {{projectname|capitalize}}
+## Contributing to {% if prettyname %}{{prettyname}}{% else %}{{projectname|capitalize}}{% endif %}
```

Comment:
Seems cumbersome. Is there a way to default init `prettyname` to `projectname|capitalize`?

Reply:
I don't know how?

Reply:
You could use [Prompt templating](https://copier.readthedocs.io/en/stable/configuring/#prompt-templating)
```.yaml
prettyname:
  default: "{{projectname|capitalize}}"
  type: str
  help: What is your project name in title case?
```
And then rely on `prettyname` to be set to avoid the `if`s everywhere.

Reply:
Thanks, works!